test/dispatchers/dispatcherTests.ts

```diff
@@ -205,6 +205,67 @@ describe("Dispatchers", () => {
       target.remove();
     });
 
+    it("onMouseDown()", () => {
+      var targetWidth = 400, targetHeight = 400;
+      var target = generateSVG(targetWidth, targetHeight);
+      // HACKHACK: PhantomJS can't measure SVGs unless they have something in them occupying space
```

Comment:
Can we put some sort of indicator that can tell us when we can remove the hackhack?